src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToSubmitHandler.java

```diff
@@ -103,27 +107,30 @@ private void processIfIncludeEvidence(SscsCaseData caseData, PreSubmitCallbackRe
             List<DwpDocument> dwpDocuments = new ArrayList<>();
 
             for (AudioVideoEvidence audioVideoEvidence : caseData.getAudioVideoEvidence()) {
```

Comment:
Do you need to iterate through the list now? If you know which one was selected through the 'selectedAudioVideoEvidence' object, can't you just use that?